offchain-modules/packages/x/src/handlers/ckb.ts

```diff
@@ -6,20 +6,21 @@ import { RPC } from '@ckb-lumos/rpc';
 import { BigNumber } from 'bignumber.js';
 import { Reader } from 'ckb-js-toolkit';
 import * as lodash from 'lodash';
+import { TransferOutSwitch } from '../audit/switch';
```

Comment:
It's not switch for ckbUnlock, we can ignore switch for now, and add that feature later

Reply:
removed this import